### PR TITLE
Adding tests for [OZ] ERC-165

### PR DESCRIPTION
**Pull request by stefan-stefanooov**
**Description**:

**Related issue(s)**:
OZ 5.x Contract Support #504

### CODE REVIEW

**Comment by quiet-node** [comment]
Hello @stefan-stefanooov, on [this branch here](https://github.com/hashgraph/hedera-smart-contracts/tree/536-Upgrade-%40openzeppelin/contracts-npm-package-to-latest-5.x-) I have upgraded OZ contracts to v5.x and modified contracts to adapt new changes. Though there's still only one failing test suite, you should create another branch off from the branch I provided above so the project can compile

**Comment by stefan-stefanooov** [comment]
> Hello @stefan-stefanooov, on [this branch here](https://github.com/hashgraph/hedera-smart-contracts/tree/536-Upgrade-%40openzeppelin/contracts-npm-package-to-latest-5.x-) I have upgraded OZ contracts to v5.x and modified contracts to adapt new changes. Though there's still only one failing test suite, you should create another branch off from the branch I provided above so the project can compile and the tests can pass

@quiet-node Thanks Logan! 
I saw you branch ..but its still work in progress so I decided to wait for you. 
What I did was just add few changes to meka the project compilable so I can continue ... 
Thanks I will rebase my work on top of yours. 
Cheers !

**Review by konstantinabl — DISMISSED** [review]

File: test/solidity/oz/erc165/erc165.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    climberSelectorContract = await ClimberSelectorFactory.deploy()
+
+    climberInterface = new ethers.utils.Interface([
+        "function hasHarness()",
+        "function hasChalk()",
+        "function hasClimbingShoes()"
+    ]);
+  })
+
+  it('should confirm supprot for: ERC-165', async function () {
```
Comment (Nana-EC) [inline test/solidity/oz/erc165/erc165.js:45]:
```suggestion
  it('should confirm support for: ERC-165', async function () {
```

File: test/solidity/oz/erc165/erc165.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    const selector = climberSelectorContract.calculateSelector()
+    const supports = await contract.supportsInterface(selector)
+    expect(supports).to.equal(true)
+  })
+
+  it('should confirm supprot for: ERC-165 -> Selector not suported', async function () {
```
Comment (Nana-EC) [inline test/solidity/oz/erc165/erc165.js:51]:
```suggestion
  it('should confirm support for: ERC-165 -> Selector not supported', async function () {
```

**Review by Nana-EC — COMMENTED** [review]
Looking good, minor items to fix

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**